doc/sphinx/proof-engine/detailed-tactic-examples.rst

```diff
@@ -226,11 +230,15 @@ name. A term is either an application of:
 
 Once we have this datatype we want to do proofs on it, like weakening:
 
-.. coqtop:: in undo
+.. coqtop:: in
```

Comment:
Why remove this `undo` from here? It didn't work? When it does work, it seems cleaner to use `.. coqtop:: undo` rather than `Abort.`

Reply:
It seemed to me it didn't, but maybe I oversaw something. Not a big loss though.